prebuilt-config/dockerfile.template

```diff
@@ -1 +1 @@
-FROM bitovi/bitops:{{ tags.bitops_base | default("dev-base") }}
+FROM bitovi/bitops:{{ tag[0] | replace('v', '') }}{{ tag[1:] }}
```

Comment:
FYI what this does, - removes the `v` prefix from `v1.2.3` in the Docker tag during templating